web-docs/concepts/opentelemetry.md

```diff
@@ -0,0 +1,126 @@
+# OpenTelemetry
+
+flagd provides telemetry data out of the box. This telemetry data is compatible with OpenTelemetry.
+
+By default, the Prometheus exporter is used for metrics which can be accessed via the `/metrics` endpoint. For example,
+with default startup flags, metrics are exposed at `http://localhost:8014/metrics`.
+
+Given below is the current implementation overview of flagd telemetry internals,
+
+![flagd telemetry](../images/flagd-telemetry.png)
+
+## Metrics
+
+flagd expose following metrics,
+
+* `http_request_duration_seconds`
+* `http_response_size_bytes`
+* `http_requests_inflight`
+* `impressions`
+* `reasons`
```

Comment:
```suggestion
* `http.server.duration`
* `http.server.response.size`
* `http.server.active_requests`
* `feature_flag.flagd.impression`
* `feature_flag.flagd.evaluation.reason`
```
I've updated the original docs also to reflect this change.

https://github.com/open-feature/flagd/edit/main/docs/configuration/flagd_telemetry.md